src/rastervision/labels/object_detection_labels.py

```diff
@@ -62,8 +63,8 @@ def from_boxlist(boxlist):
             boxlist.get(), boxlist.get_field('classes'), scores)
 
     @staticmethod
-    def from_geojson(geojson, crs_transformer):
-        return geojson_to_labels(geojson, crs_transformer)
+    def from_geojson(geojson, crs_transformer, extent):
```

Comment:
Should `extent` be required at a higher level? It seems a bit off to require extent for GeoJSON labels, but not for object detection labels in general. Or, perhaps the geojson labels shouldn't require an extent and this functionality be solved a different way. For example, will there be times we want to read in a set of geojson labels, where we don't have an extent? Maybe not now, but I could see that happening in the future - and we don't want to require someone to make a false extent to get use out of the class. I'd be in favor of just having call sites that require the subset of labels that overlap an extent to explicitly call `labels.get_intersection(extent)`, rather then bake in the requirement of all labels (or, in this case, only a subclass of OD labels) always having an extent passed in.

Reply:
I see what you mean. But, we're already passing in the crs_transformer to the label_store (which is associated with the raster_source), so it doesn't seem like much of a stretch to also pass in the extent. Doing it this way also avoids potential future bugs where people forget to do the get_overlap operation when its needed. So I'm leaning towards leaving it as is.

Reply:
OK. TBH, I have had the same sort of issue with having to pass in a crs_transformer everywhere - there has been occasion (though I can't remember the specific instance) where I wanted to new up an object but was required to pass in a crs_transformer, which I didn't have. This type of coupling makes it harder to utilize things in new situations as time goes on, and this would put us further down that path. Then again, I'm not that strongly against it. Let's just be on the lookout for dependency couplings that cause pain in the future, and perhaps refactor them away later.

Reply:
I see how the dependence of LabelStore on RasterSources is annoying, but I don't see how we can decouple them, since you need to know the CRS of the rasters to convert from map coordinates in the JSON files to the pixel coordinates used by the LabelStore. We can talk about this in person, but for now I'll just merge this.